jetty-core/jetty-io/src/main/java/org/eclipse/jetty/io/Content.java

```diff
@@ -902,9 +903,12 @@ default Chunk asReadOnly()
         interface Processor
         {
             /**
-             * @param chunk The chunk to be considered for processing.
+             * @param chunk The chunk to be considered for processing, including persistent and transient failures.
              * @param callback The callback that will be called once the accepted chunk is processed.
-             * @return True if the chunk will be process and the callback will be called (or may have already been called), false otherwise.
+             *                 {@link Callback#succeeded() Succeeding} this callback will allow the processing of subsequent chunks.
+             *                 {@link Callback#failed(Throwable) Failing} this callback will fail the processing of all chunks.
+             * @return {@code True} if the chunk will be processed asynchronously and the callback will be called (or may have already been called),
+             *         {@code false} otherwise, in which case subsequent chunks may be processed and the passed callback ignored.
```

Comment:
```suggestion
             *         {@code False} otherwise, in which case subsequent chunks may be processed and the passed callback ignored.
```

Or alternatively make both line 910 aand 911 lowercase

Reply:
My preference is both lower case.